docs/reference/node/telemetry-config.md

```diff
@@ -0,0 +1,42 @@
+title: Telemetry
+
+#Initialization
+
+When a node is run using the algod command, before the script starts the server, it configures its telemetry based on the appropriate logging.config file. The algoh command, which hosts algod, configures logging and telemetry before calling algod. Both of these commands can override the config file’s telemetry enable field’s value using the -t flag.
+When a node’s telemetry is enabled, a telemetry state (which wraps the node’s hook for the elasticsearch server to which the logs are saved) is added to the node’s logger reflecting the fields contained within the appropriate config file.
+
+#Configuration
+
+A node’s telemetry status can be managed using the diagcfg CLI, which modifies the node’s logging.config file. This file instructs the node’s construction of its TelemetryConfig struct, defining the following fields:
```

Comment:
Could you see if you could format this into a table-ish, with three columns - configuration key, data type, and description ? it would greatly improve the readability.